### ISSUE TITLE
[Feature request] Cache results

### ISSUE THREAD

**Issue by mattruzzi** [useful-forks/useful-forks.github.io#39]
It would be great if there was a way to cache the results since they take a long time to generate and require a lot of API calls.

**Comment by payne911** [comment]
Yeah, I'm aware. The `Export CSV` button was introduced as a quick-fix to this specifically.

**Comment by mattruzzi** [comment]
Could the results be cached in the local browser cache?

**Comment by payne911** [comment]
Essentially, yes.
Maybe `octokit` somehow already has some caching capability: that would have to be investigated.
But all in all, given the limited time I can give to this project, I'm not sure I will personally explore how to implement this feature in the near future. Others are welcome to try, though!

The main use-case I can see this being useful for is when the search ends up being interrupted, and one has to refresh the page to start again: if all calls were cached, it would almost immediately restart from where it left off.
This mostly only happens on enormous scans, and would probably represent a decent amount of coding, though.
